### PR TITLE
Add analytics

### PR DESCRIPTION
**Pull request by segun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
         "query-string": "^7.1.1",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
+        "react-ga": "^3.3.1",
+        "react-ga4": "^1.4.1",
```
Comment (Adarsh-K) [inline package.json:25]: Why are 2 different versions required?

File: src/components/send-card/send-card.tsx [hunk 2]
```diff
@@ -41,6 +44,7 @@ export const SendCard = () => {
 
     const handleNetworkChange = (event: ChangeEvent<HTMLInputElement>) => {
         setNetwork(event.target.value);
+        eventTracker("Network Change", event.target.value);
         if (event.target.value === "Gnosis Chain") {
```
Comment (Adarsh-K) [inline src/components/send-card/send-card.tsx:48]: Better to extract `event.target.value` in a variable as it's used 3 times here.

File: src/components/send-card/send-card.tsx [hunk 3]
```diff
@@ -80,7 +84,7 @@ export const SendCard = () => {
         });
     };
 
-    const sendRequest = async () => {
+    const sendRequest = async () => {
```
Comment (Adarsh-K) [inline src/components/send-card/send-card.tsx:87]: - There shouldn't be spacing after `{`.
- Don't know how it passed linter. Maybe you wanna check if the linter is working correctly too.

File: src/components/send-card/send-card.tsx [hunk 4]
```diff
@@ -111,7 +115,7 @@ export const SendCard = () => {
                 network,
                 tweetUrl,
                 amount: _amount,
-            };
+            };
```
Comment (Adarsh-K) [inline src/components/send-card/send-card.tsx:118]: Same as above, no spacing

File: src/components/send-card/send-card.tsx [hunk 5]
```diff
@@ -135,6 +139,8 @@ export const SendCard = () => {
                     setShowLoading(false);
                     toast.error(error.response.data.data.error);
                 });
+                
+                eventTracker("Send Request", JSON.stringify(req));
```
Comment (Adarsh-K) [inline src/components/send-card/send-card.tsx:143]: It has an additional tab spacing. Please check for the linter's correctness.

**Review by Adarsh-K — CHANGES_REQUESTED** [review]

**Comment by Adarsh-K** [comment]
@segun Let's add the **eventTracker** in the following functions in `send-card.tsx`:

- [ ] handleTweetUrlChange
- [ ] handleAmountChange
- [ ] onVerifyCaptcha
- [ ] paste

File: package.json (hunk 1 above)
Comment (segun) [reply package.json:25]: Moving to ga4 eventually, but yeah not needed here.

File: src/components/send-card/send-card.tsx (hunk 2 above)
Comment (segun) [reply src/components/send-card/send-card.tsx:48]: fun fact, in solidity, you're better off not extracting this into a variable as every operation costs you $$$. ;)

**Pull request merged**